### ISSUE TITLE
Possibility of adding a working demo code

### ISSUE THREAD

**Issue by sumeetssaurav** [google-research/morph-net#22]
Hello,
Thank you for such a wonderful work and open-sourcing it. However, if a working demo code could be added, then naive users like me would find it more comfortable using the library. Therefore, could you please add some working demo code of experiments mentioned in the paper using some  seed network.

**Comment by ayp-google** [comment]
You can find example networks here:
https://github.com/tensorflow/models/tree/master/research/slim/nets

There you can find networks such as Inception and Resnet family of architectures.  Once you are familiar with training those networks, you can follow the MorphNet guide to try out MorphNet.  You can check out [TF-slim](https://github.com/tensorflow/models/blob/master/research/slim/slim_walkthrough.ipynb) for a working example in a Python notebook.

**Comment by Aspirinkb** [comment]
I don't think one is willing to use slim to try MorphNet...

**Comment by eladeban** [comment]
@Aspirinkb Interesting point :)
Could you suggest a specific model (and open source code for it) that **you** will be happy to see the MorphNet demo for.

**Comment by Aspirinkb** [comment]
@eladeban Considering Keras is so popular and has a close relation with TensorFlow, so I think a model in Keras is a good start. What do you think ResNet50?

**Comment by waqarahmed4544** [comment]
> @eladeban Considering Keras is so popular and has a close relation with TensorFlow, so I think a model in Keras is a good start. What do you think ResNet50?

For instance, **cifar10_resnet.py** trains a ResNet on the CIFAR10 available at
https://github.com/keras-team/keras/tree/master/examples
I will be interested to see a modified version of cifar10_resnet.py that proposes a new model by adjusting the number of output channels in each convolution layer.

**Comment by 010akv** [comment]
(Also related to #41 ). A demo with a detection model from https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md would be very helpful in addition to classification demo. I'd pick `ssd_mobilenet_v1_0.75_depth_quantized_coco` if I were to suggest one that I'd be happy to see MorphNet demo for.

**Comment by JMFlin** [comment]
Any update on a keras resnet example?

**Comment by eladeban** [comment]
Sorry about the delay, but we don't have the bandwidth to do these examples. We will be more than happy to show case such examples here if somewhat is willing to take the task.

**Comment by ferova** [comment]
In case anyone is still interested (a year later), I have been working on creating examples for this really cool library, hoping that more people make use of it. 

The tutorials are [here](https://github.com/ferova/MorphNetTutorials), for now I only have created one and am working on a couple more. If anyone's interested in seeing more of this or has any comments please contact me or open an issue in that repo.
